src/components/v5/shared/Avatar/Avatar.tsx

```diff
@@ -46,7 +48,19 @@ const Avatar: FC<PropsWithChildren<AvatarProps>> = ({
       title={title}
     >
       {source ? (
-        <div className={styles.image} style={imageStyle} />
+        <div
+          className={clsx(
+            'w-full h-full absolute top-0 left-0 rounded-full bg-cover border-2',
+            {
+              'border-purple-400': mode === 'top',
+              'border-warning-400': mode === 'active',
+              'border-blue-400': mode === 'dedicated',
+              'border-green-400': mode === 'new',
+              'border-none': mode === 'general' || mode === undefined,
```

Comment:
You can set a default mode in line 20 and then  mode === undefined shouldn't be needed

Reply:
the mode sometimes returns undefined ( it depends on API ) and in that case the border should be set as none, so I can't set as default mode then all members without mode will have border what is incorrect

Reply:
You can set mode as `general` by default, it will add the same styles and you won't have to set a fallback in other places then

Reply:
fixed